.github/workflows/test.yml

```diff
@@ -317,7 +321,9 @@ jobs:
         path: /Users/runner/Library/Python/3.11/lib/python/site-packages
         key: metal-m1-testing-user3-packages-${{ hashFiles('**/setup.py') }}
     - name: Install Dependencies
-      run: pip install --user -e '.[testing]' --extra-index-url https://download.pytorch.org/whl/cpu
+      run: |
+        pip install --user -e '.[testing]' --extra-index-url https://download.pytorch.org/whl/cpu
```

Comment:
Why is this changing

Reply:
There is an issue with torchtune and setuptools (https://github.com/pytorch/torchtune/issues/1111), when add torchtune in setup.py under testing the NV and GPU test fail with the same error of Import packaging. It can be fixed installing torchtune nightly or downgrading setuptools to 69.0.0, in this case I install nightly

Reply:
Why is this in this PR? Should be unrelated